controller/src/enhancements/bomb.rs

```diff
@@ -148,29 +190,61 @@ impl BombInfo {
     }
 }
 
-/// % of the screens height
-const PLAYER_AVATAR_TOP_OFFSET: f32 = 0.004;
-
-/// % of the screens height
-const PLAYER_AVATAR_SIZE: f32 = 0.05;
-
 impl Enhancement for BombInfo {
-    fn update(&mut self, ctx: &crate::UpdateContext) -> anyhow::Result<()> {
+    fn update(&mut self, ctx: &UpdateContext) -> anyhow::Result<()> {
         if !ctx.settings.bomb_timer {
             return Ok(());
         }
 
+        for entity_identity in ctx.cs2_entities.all_identities() {
+            let entity_class = ctx
+                .class_name_cache
+                .lookup(&entity_identity.entity_class_info()?)?;
+
+            if !entity_class
+                .map(|name| *name == "C_CSPlayerPawn")
+                .unwrap_or(false)
+            {
+                /* entity is not a player pawn */
+                continue;
+            }
+
+            let player_pawn = entity_identity
+                .entity_ptr::<C_CSPlayerPawn>()?
+                .read_schema()
+                .unwrap();
```

Comment:
Please do not use `unwrap` as it will panic when an error occurs.  
Use `?` in combination with `.context("my message here")`

Reply:
Alright